### PR TITLE
Add test to cover translator comments with tab indentations

### PR DESCRIPTION
**Pull request by ocean90**
Fixes https://github.com/wp-cli/i18n-command/issues/169.

The test will pass with the fix from https://github.com/oscarotero/Gettext/pull/215.

### CODE REVIEW

File: features/makepot.feature [hunk 1]
```diff
@@ -520,9 +522,14 @@ Feature: Generate a POT file of a WordPress project
...
        __( 'off', 'foo-plugin' );
 
-       /* translators: this should get extracted. */ $foo = __( 'baba', 'foo-plugin' );
+      /* translators: this should get extracted. */ $foo = __( 'baba', 'foo-plugin' );
 
-       /* translators: boo */ /* translators: this should get extracted too. */ /* some other comment */ $bar = g ( __( 'bubu', 'foo-plugin' ) );
+      /* translators: boo */ /* translators: this should get extracted too. */ /* some other comment */ $bar = g( __( 'bubu', 'foo-plugin' ) );
+
+      {TAB}/*
+      {TAB} * translators: this comment block is intend with a tab and should get extracted too.
```
Comment (schlessera) [inline features/makepot.feature]:
```suggestion
      {TAB} * translators: this comment block is indented with a tab and should get extracted too.
```

File: features/makepot.feature [hunk 2]
```diff
@@ -568,6 +575,10 @@ Feature: Generate a POT file of a WordPress project
       """
       #. translators: this should get extracted too.
       """
+    And the foo-plugin/foo-plugin.pot file should contain:
+      """
+      #. translators: this comment block is intend with a tab and should get extracted too.
```
Comment (schlessera) [inline features/makepot.feature]:
```suggestion
      #. translators: this comment block is indented with a tab and should get extracted too.
```

**Comment by swissspidy** [comment]
@schlessera Are you fine with referencing `dev-master#68d9f76f709a12ad7c3159d0acb08f9cad159426` of the upstream library directly, or should we wait for a release?

**Comment by ocean90** [comment]
The reference should be just for testing, we should wait for the release. The current way generates a warning during `composer validate` and `composer install`.

**Comment by schlessera** [comment]
I'm fine with referencing the specific commit, provided that we add corresponding issues to make sure we don't forget to change it back to a release again later.

**Comment by swissspidy** [comment]
Version 4.6.3 of the gettext library was just released.

@ocean90 Would you mind updating your PR? Thanks!

**Pull request merged**